crates/router/src/connector/stripe/transformers.rs

```diff
@@ -4025,6 +4028,40 @@ impl ForeignTryFrom<(&Option<ErrorDetails>, u16, String)> for types::PaymentsRes
     }
 }
 
+pub(super) fn transform_headers_for_connect_platform_payment(
+    charges: Option<&PaymentCharges>,
+    header: &mut Vec<(String, services::request::Maskable<String>)>,
+) {
+    charges.map(|charge| match &charge.charge_type {
```

Comment:
you can avoid inner if statement, by matching enum in match expression itself. `if let` instead of match possible here?